wrapper.go

```diff
@@ -168,7 +168,11 @@ func Decode(orig, recovery [][]byte) (decoded [][]byte, err error) {
 	}
 	for i := 0; i < len(recovery); i++ {
 		if recovery[i] != nil {
-			decoded[i+len(orig)] = recovery[i]
+			offset := 0 // special case: 1 share
+			if len(orig) > 1 {
+				offset = len(orig)
+			}
+			decoded[i+offset] = recovery[i]
```

Comment:
This doesn't look right. If there's one original share and one parity share, then line 175 will assign `decoded[0] = recovery[0]` then terminate. The value here may be correct (polynomial interpolation of a single point is a single point?), but it doesn't address the root cause. `decoded` should have length `2`, not length `1`, and therefore does not address #14.

Reply:
> decoded should have length 2, not length 1, and therefore this change does not fix the root cause of https://github.com/celestiaorg/go-leopard/issues/14 even if it hides the symptoms.

hmm, maybe but that is not what the underlying leopard library returns in that case:

https://github.com/catid/leopard/blob/22ddc7804998d31c8f1a2617ee720e063b1fa6cd/leopard.cpp#L207-L208

Reply:
Let me dig into this again.

Reply:
> (polynomial interpolation of a single point is a single point, the point itself??)

Also, yes. See what happens on encoding if len(original data) == 1:
https://github.com/catid/leopard/blob/22ddc7804998d31c8f1a2617ee720e063b1fa6cd/leopard.cpp#L143-L149